src/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface ISafeKeyboardDetector {
+  isCurrentKeyboardSafe: (customAllowedKeyboardList?: string[]) => boolean;
```

Comment:
on n'avait pas dit avec Cyril que `customAllowedKeyboardList` pouvait être dans un fichier de config ? enfin en vrai les 2 me vont mais chaud de savoir s'il y a eu des discussions entre qui validait l'utilisation d'un param de fonction plutôt

Reply:
C'est moi qui ai challengé la reco d'avoir une API séparée pour choisir la liste.

La reco ici, is tu veux une liste de claviers personnalisées, c'est de wrapper cette fonction dans ton code. Ca évite d'avoir un autre endroit sans lien apparent dans le code qui impacte le résultat de cette fonction